sway/desktop/launcher.c

```diff
@@ -174,26 +184,21 @@ void root_record_workspace_pid(pid_t pid) {
 		return;
 	}
 
-	struct timespec now;
-	clock_gettime(CLOCK_MONOTONIC, &now);
-
-	// Remove expired entries
-	static const int timeout = 60;
-	struct pid_workspace *old, *_old;
-	wl_list_for_each_safe(old, _old, &pid_workspaces, link) {
-		if (now.tv_sec - old->time_added.tv_sec >= timeout) {
-			pid_workspace_destroy(old);
-		}
-	}
-
 	struct pid_workspace *pw = calloc(1, sizeof(struct pid_workspace));
+	struct wlr_xdg_activation_token_v1 *token =
+		wlr_xdg_activation_token_v1_create(server.xdg_activation_v1);
```

Comment:
It seems like `server.xdg_activation_v1` is used here, but doesn't exist yet at this point in the Git history?

Reply:
I'm mistaken, we already have this in master for urgency.